content/documentation/use-networks.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Use Networks
+description: Use Testnet | Iron Fish Documentation
+---
+
+Iron Fish has 3 networks built into the SDK by default. These networks are useful for development and testing rather than trying to wait for someone to send you Mainnet $IRON.
+
+> **Note:** Once you start a node on a certain network, that node cannot be used on another network. Delete the data directory in `ironfish browse` to reset the data directory.
```

Comment:
Should we also refer to `ironfish reset`?

Reply:
I just tested that and it doesn't actually work to clear the right flags to let me switch networks. We should fix that, I'll file a linear issue.

Reply:
Ok fixed